helpers/test/plugins.test.js

```diff
@@ -0,0 +1,67 @@
+import { getPlugin, getPlugins, registerPlugin } from '../plugins.js';
+import { expect } from '@brightspace-ui/testing';
+
+describe('plugins', () => {
+
+	before(() => {
+		registerPlugin('test-plugins', { prop1: 'beer' });
+		registerPlugin('test-plugins', { prop1: 'donuts' });
+		registerPlugin('test-plugins-sorted', { prop1: 'beer' }, { sort: 2 });
+		registerPlugin('test-plugins-sorted', { prop1: 'donuts' }, { sort: 1 });
+		registerPlugin('test-plugins-keyed', { prop1: 'beer' }, { key: 'plugin1' });
+		registerPlugin('test-plugins-keyed', { prop1: 'donuts' }, { key: 'plugin2' });
+	});
```

Comment:
Would it make sense to export a `reset` method that empties everything out? That way you could call it in `afterEach` and then each test could be more isolated.

Reply:
Haha, I actually had that method while I was writing the tests, but then I thought to myself... if someone imported that and called it in the app, it would be very bad. But yeah, I'm happy to add that is we prefer. 😄

Reply:
Added this back, and organized tests a bit.

Reply:
Cool. Yeah I think it's fine -- I'd love to switch core over to the `exports` model and then we could actually hide it from consumers.